### PR TITLE
Wizard Organ Damage

### PR DESCRIPTION
**Pull request by Geevies**
* Wizard healing spells now heal internal organs as well.

This is because brainmed focuses more on organs than external damage.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/modules/spell_system/spells/spell_list/targeted_spells.dm [hunk 1]
```diff
@@ -142,4 +144,11 @@ Targeted spells have two useful flags: INCLUDEUSER and SELECTABLE. These are exp
...
-	target.stuttering += amt_stuttering
\ No newline at end of file
+	target.stuttering += amt_stuttering
+	if(ishuman(target))
+		var/mob/living/carbon/human/H = target
+		for(var/obj/item/organ/O in H.internal_organs)
+			if(amt_organ > 0)
+				O.take_damage(amt_organ)
+			else
+				O.heal_damage(amt_organ)
```
Comment (Arrow768) [inline code/modules/spell_system/spells/spell_list/targeted_spells.dm:154]: Pretty sure that doesnt work as amt_organ is negative and these procs expect a positive value

File: code/modules/spell_system/spells/spell_list/targeted_spells.dm [hunk 2]
```diff
@@ -142,4 +144,12 @@ Targeted spells have two useful flags: INCLUDEUSER and SELECTABLE. These are exp
...
\ No newline at end of file
+	target.stuttering += amt_stuttering
+	if(ishuman(target))
+		var/mob/living/carbon/human/H = target
+		for(var/obj/item/organ/O in H.internal_organs)
+			if(amt_organ > 0)
+				O.take_damage(amt_organ)
+			else
+				amt_organ *= -1
+				O.heal_damage(amt_organ)
```
Comment (Arrow768) [inline code/modules/spell_system/spells/spell_list/targeted_spells.dm:155]:
```suggestion
				O.heal_damage(-amt_organ)
```
Wouldnt that work aswell?

**Comment by Geevies** [comment]
!review

**Pull request merged**